docs/filters.md

```diff
@@ -227,50 +232,72 @@ Meta-qualifier. Changes the semantics of search terms to check only either the t
 
 Matches issues of a given issue type. Pull requests are loosely considered issues in that the same operations work on both; this predicate allows users to distinguish them.
 
+Aliases: `ty`
+
+- `issue` can be written as `i`
+- `pr` can be written as `pullrequest` or `p`
+
 ### is
 
 *Expects one of `open`, `closed`, `pr`, `issue`, `merged`, `unmerged`, `read`, `unread`*
 
 Matches issues which are either open or closed, a pull request or an issue, depending on their merged status if they are pull requests, or read or unread. Is partially an alias for `state` and `type`.
 
+Aliases: 
+
+- `merged` can be written as `mg`
+- `unmerged` can be written as `um`
+- `read` can be written as `rd`
+- `unread` can be written as `ur`
+
 ### created
 
 *Expects a date or date range*
 
 Matches issues which were created on a given date, or within a given date range.
 
+Aliases: `cr`
+
 ### updated
 
 *Expects a number or  number range*
 
 Matches issues which were updated in the given number of hours. For example, `updated:<24` would match issues updated in the last day. If a number `n` is given, it is implicitly translated to `<n`. Number ranges are written using a relational operator (.e.g `>5`, `<=10`).
 
+Aliases: `u`
+
 ### repo
 
 *Expects a repo id*
 
 Matches issues of the given repository. If omitted, will match issues of the default repository instead (not all repositories).
 
+Aliases: `r`
+
 ### sort
 
 *Expects a comma-separated list of sorting criteria. For example, `repo, ~updated, -comments`.*
 
 Sorts a repository by the list of criteria, going from left to right. Negated criteria will reverse the ordering that they describe.
 
+Aliases: `s`
+
 ### count
 
 *Expects a number*
 
 The maximum number of issues that can be displayed in a panel. For example, `count:4` would display a maximum of 4 issues in the panel.
 
+Aliases: `cn`
+
 Available sorting criteria:
 
-- `comments`: sorts by number of comments (in descending order)
+- `comments`: sorts by number of comments (in descending order). Aliases: `cm`.
```

Comment:
Could you help to shift this back under `sort` heading? Seems to have been wrongly moved down here.